### PR TITLE
Contest create UI

### PR DESCRIPTION
**Pull request by mctran-phi**
### What this PR does (required):
- A form that allows users to create a contest.

### Screenshots / Videos (required):
- Screen Shot 2021-07-14 at 7 23 29 PM

### Any information needed to test this feature (required):
- Create a route path to the page.

### Any issues with the current functionality (optional):
- Did not implement a handleSubmit, wasn't sure where I'm submitting the information.
- I wasn't able to implement an onClick select image as well.

### CODE REVIEW

**Comment by tsbolty** [comment]
I like it a lot. We can write the onclick when the time comes.

File: client/src/pages/CreateContest/CreateContestForm/CreateContestForm.tsx [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+                </Grid>
+              </Grid>
+            </Grid>
+          </Grid>
+        </Box>
+        <Box mt={3} mb={3} className={classes.box}>
+          <Typography className={classes.label}>Which designs do you like?</Typography>
+          <Box mt={3} mb={3} className={classes.imageList}>
+            <ImageList rowHeight={160} cols={4} className={classes.images}>
+              {images.map((image: string) => (
```
Comment (rajivtitus) [inline client/src/pages/CreateContest/CreateContestForm/CreateContestForm.tsx:124]: 👍🏽

File: client/src/pages/CreateContest/useStyles.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles(() => ({
+  root: {
```
Comment (rajivtitus) [inline client/src/pages/CreateContest/useStyles.ts:4]: Please remove empty styles

File: client/src/pages/CreateContest/CreateContestForm/CreateContestForm.tsx [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+import AttachMoneyIcon from '@material-ui/icons/AttachMoney';
+import DateFnsUtils from '@date-io/date-fns';
+import useStyles from './useStyles';
+
+export default function CreateContestForm():JSX.Element {
+  const [design, setDesign] = useState<string>('');
+  const [description, setDescription] = useState<string>('');
+  const [prize, setPrize] = useState<string>('');
+  const [date, setDate] = useState<Date | null>(new Date());
+  const [time, setTime] = useState<Date | null>(new Date());
```
Comment (rajivtitus) [inline client/src/pages/CreateContest/CreateContestForm/CreateContestForm.tsx:16]: We can mange the date and time using the `date` state, we don't to have a separate state for time :)

**Review by rajivtitus — APPROVED** [review]
Good work, UI looks good too!

Just a couple of comments, please have a look and address. Thanks

**Pull request closed without merging**